.github/workflows/push.yml

```diff
@@ -19,13 +19,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-
+      - name: Set env
+        run: echo "TAGS=kintohub/kinto-dashboard:${GITHUB_SHA},kintohub/kinto-dashboard:latest" >> $GITHUB_ENV
```

Comment:
can u explain what this is doing?

Reply:
sending the tags (latest + sha) to the next step.
somehow it was not woking if I wrote like this.
```
    tags: "kintohub/kinto-dashboard:${GITHUB_SHA},kintohub/kinto-dashboard:latest"
```

Reply:
Understood